wallet/internal/txsizes/size.go

```diff
@@ -68,6 +68,26 @@ const (
 	//   - OP_CHECKSIG
 	P2PKHPkScriptSize = 1 + 1 + 1 + 20 + 1 + 1
 
+	// P2SHPkScriptSize is the size of a transaction output script that
+	// pays to a script hash.  It is calculated as:
+	//
+	//   - OP_HASH160
+	//   - OP_DATA_20
+	//   - 20 bytes script hash
+	//   - OP_EQUAL
+	P2SHPkScriptSize = 1 + 1 + 20 + 1
+
+	// TicketCommitmentScriptSize is the size of a ticket purchase commitment
+	// script. It is calculated as:
+	//
+	//   - OP_RETURN
+	//   - OP_DATA_20
+	//   - 20 bytes P2SH/P2PKH
+	//   - 8 byte amount
+	//   - 4 byte fee range limits
```

Comment:
Fee range limit is 2 bytes long:
https://github.com/decred/dcrd/blob/master/txscript/standard.go#L963